### PR TITLE
W-15807150-otel-traces-rtf-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by Crispy-Salesforce — DISMISSED** [review]

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 1]
```diff
@@ -121,3 +123,24 @@ anypoint.platform.config.analytics.agent.enabled=false
 
 [NOTE]
 For compliance or geospecific requirements if your team or organization cannot send metrics to Anypoint control plane, you can disable Anypoint metrics if you are not contractually obligated to send these metrics for Anypoint usage and billing purposes.
+
+[[enable-trace-data]]
+== Enable Trace Data Collection
+
```
Comment (sarathecoubian) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc:129]: [NOTE]
include::reuse::partial$billing/pricing.adoc[tag=featureAdvReq]

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 2]
```diff
@@ -121,3 +123,24 @@ anypoint.platform.config.analytics.agent.enabled=false
...
+Traces enable you to understand the execution path of a transaction in MuleSoft ecosystem by having full visibility into the execution path of API calls or application transactions. This feature exposes tracing information in OpenTelemetry standard from Mule runtime into the Anypoint Monitoring xref:monitoring::telemetry-exporter.adoc[Telemetry Exporter].
```
Comment (sarathecoubian) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc:130]: Traces enable you to understand the path of a transaction in the MuleSoft ecosystem by providing full visibility into the execution path of API calls or application transactions. 
When you enable trace data collection for an app, Anypoint Platform collects and exposes tracing information in the OpenTelemetry standard from Mule Runtime, enabling you to export it using the Anypoint Monitoring xref:monitoring::telemetry-exporter.adoc[Telemetry Exporter].

Comment (sarathecoubian) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc]:
```suggestion
Traces enable you to understand the path of a transaction in the MuleSoft ecosystem by providing full visibility into the execution path of API calls or application transactions. 
When you enable trace data collection for an app, Anypoint Platform collects and exposes tracing information from Mule runtime engine in the OpenTelemetry standard, enabling you to export it using the Anypoint Monitoring xref:monitoring::telemetry-exporter.adoc[Telemetry Exporter]. 
```

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 3]
```diff
@@ -121,3 +123,24 @@ anypoint.platform.config.analytics.agent.enabled=false
...
+. Select the *Enable tracing* checkbox.
+. Click *Deploy Application* if you are enabling the feature for a new deployment, or *Apply Changes* if you are editing an existing application.
+
+[NOTE]
+You can't enable trace data collection via Mule Maven plugin, Anypoint CLI, or Studio yet. You can enable trace data collection via UI and API.
+
+Once you enable tracing, and redeploy your application, the Runtime Manager backend starts instrumenting traces for your application.
+
+[NOTE]
+include::reuse::partial$billing/pricing.adoc[tag=featureAdvReq]
```
Comment (sarathecoubian) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc]:
```suggestion
```

File: modules/ROOT/pages/use-anypoint-monitoring.adoc (hunk 1 above)
Comment (sarathecoubian) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc:129]:
```suggestion

[NOTE]
include::reuse::partial$billing/pricing.adoc[tag=featureAdvReq]

```

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 4]
```diff
@@ -121,3 +123,24 @@ anypoint.platform.config.analytics.agent.enabled=false
...
+You can enable trace data collection for your Runtime Fabric applications running on Mule runtime version 4.6 or later. 
+
+To enable distributed tracing in your application deployments:
+
+. Click the *Monitoring* tab.
+. Select the *Enable tracing* checkbox.
+. Click *Deploy Application* if you are enabling the feature for a new deployment, or *Apply Changes* if you are editing an existing application.
+
+[NOTE]
+You can't enable trace data collection via Mule Maven plugin, Anypoint CLI, or Studio yet. You can enable trace data collection via UI and API.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc]:
```suggestion
```

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 5]
```diff
@@ -121,3 +123,24 @@ anypoint.platform.config.analytics.agent.enabled=false
 
 [NOTE]
 For compliance or geospecific requirements if your team or organization cannot send metrics to Anypoint control plane, you can disable Anypoint metrics if you are not contractually obligated to send these metrics for Anypoint usage and billing purposes.
+
+[[enable-trace-data]]
+== Enable Trace Data Collection
+
+Traces enable you to understand the execution path of a transaction in MuleSoft ecosystem by having full visibility into the execution path of API calls or application transactions. This feature exposes tracing information in OpenTelemetry standard from Mule runtime into the Anypoint Monitoring xref:monitoring::telemetry-exporter.adoc[Telemetry Exporter].
+
+You can enable trace data collection for your Runtime Fabric applications running on Mule runtime version 4.6 or later.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc]:
```suggestion
You can enable trace data collection for your Runtime Fabric applications running on Mule version 4.6 or later. 
You can't enable trace data collection using the Mule Maven plugin, Anypoint CLI, or Anypoint Studio. You can enable trace data collection using only the UI and API.
```

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 6]
```diff
@@ -121,3 +123,24 @@ anypoint.platform.config.analytics.agent.enabled=false
...
 For compliance or geospecific requirements if your team or organization cannot send metrics to Anypoint control plane, you can disable Anypoint metrics if you are not contractually obligated to send these metrics for Anypoint usage and billing purposes.
+
+[[enable-trace-data]]
+== Enable Trace Data Collection
+
+Traces enable you to understand the execution path of a transaction in MuleSoft ecosystem by having full visibility into the execution path of API calls or application transactions. This feature exposes tracing information in OpenTelemetry standard from Mule runtime into the Anypoint Monitoring xref:monitoring::telemetry-exporter.adoc[Telemetry Exporter].
+
+You can enable trace data collection for your Runtime Fabric applications running on Mule runtime version 4.6 or later. 
+
+To enable distributed tracing in your application deployments:
```
Comment (sarathecoubian) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc]:
```suggestion
To enable trace data collection for an app:
```

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**